ui/app/ducks/swaps/swaps.js

```diff
@@ -95,6 +95,8 @@ const slice = createSlice({
       state.approveTxId = null
       state.balanceError = false
       state.fetchingQuotes = false
+      state.customGas.limit = null
```

Comment:
Ensures that the custom modal amounts are cleared when the user goes "Back" from the view-quote screen to the build-quote screen